client/app/scripts/superdesk-authoring/views/send-item.html

```diff
@@ -50,17 +50,17 @@ <h3 translate ng-show="macros">Transformation</h3>
                     ng-disabled="!selectedDesk || (mode !== 'ingest' && selectedStage._id === item.task.stage)"
                     translate>send</button>
                 </li>
-                <li ng-show="mode === 'authoring' && canSendAndContinue()">
-                    <button class="send" ng-click="sendAndContinue()"
-                    ng-disabled="!selectedDesk || (mode !== 'ingest' && selectedStage._id === item.task.stage)"
-                    translate>send and continue</button>
-                </li>                
                 <li ng-show="mode !== 'authoring'">
                     <button class="send"
                         ng-click="send(true)"
                         ng-disabled="!selectedDesk || (mode !== 'ingest' && selectedStage._id === item.task.stage) || config"
                         translate>send and open</button>
                 </li>
+                <li ng-show="mode === 'authoring' && canSendAndContinue()">
+                    <button class="send" ng-click="sendAndContinue()"
+                    ng-disabled="!selectedDesk || (mode !== 'ingest' && selectedStage._id === item.task.stage)"
+                    translate>send and continue</button>
```

Comment:
fyi it's better to do this in html, in e2e tests it's not consistent in chrome/firefox, chrome will give you uppercase and firefox will be lowercase